R/helpers.R

```diff
@@ -38,12 +43,17 @@ md <- function(text) {
 
 #' Interpret input text as HTML-formatted text
 #'
+#' For certain pieces of text (like in column labels or table headings) we may
+#' want to express them as raw HTML. In fact, with HTML, anything goes so it can
+#' be much more than just text. The `html()` function will guard the input HTML
+#' against escaping, so, your HTML tags will come through as HTML when
+#' rendered... to HTML.
```

Comment:
Probably should add a note that it's not safe to put untrusted user input in here (like database values that can be influenced by a potentially malicious user--even a username!--or literally anything from Shiny `input`) as it'd then be possible to inject JavaScript into the end result document.

Reply:
Will file a separate issue for this change.